packages/@react-spectrum/list/src/listview.css

```diff
@@ -191,5 +191,7 @@
   justify-content: center;
   width: 100%;
   height: 100%;
+  .loadingMore {
```

Comment:
this classname isn't the classname you think it is?
I'm assuming you wanted `.react-spectrum-ListView-centeredWrapper--loadingMore`

Reply:
derp you are right, lemme fix